jboss-eap7-with-helm.yaml

```diff
@@ -86,7 +87,10 @@ spec:
         1. In the navigation menu, click [Helm]{{highlight qs-nav-helm}}.
 
         1. Click **eap74** Helm release.  
-           The Helm release details page shows all the resources that were created when the JBoss EAP 7 application was deployed.
+           This page shows information about the Helm release that you installed.  
+           In particular, The **Resources** panel lists all the resources created by this Helm release.
+
+            - Once this quickstart is complete, you can upgrade or uninstall the Helm release by using the **Actions** dropdown list.
```

Comment:
The **Helm Release details** page opens. It shows all the information related to the Helm release that you installed.  
1. Click the **Resources** tab. It lists all the resources created by this Helm release.
- You can upgrade or uninstall a Helm release by using the **Actions** dropdown list.

Reply:
> You can upgrade or uninstall a Helm release by using the Actions dropdown list.

I'm not sure about that sentence. My intent was to let the users know that *eventually* they can upgrade or uninstall the Helm release from here but not imply that they have to do it now (which would invalidate the remaining of the quickstart).

It might be better to not write anything here than making it more confusing.

(I've update the rest of the instructions)